### ISSUE TITLE
Android static libraries

### ISSUE THREAD

**Issue by MarcelKlammer** [jasonacox/Build-OpenSSL-cURL#35]
Since you are so responsive, would it be possible for you to add Android static library scripts as well? 

All libs compiled with r21 NDK would be perfect. Most other repositories around are outdated or don't work.

**Comment by jasonacox** [comment]
Thanks for the kind words!  I haven't built any Android libraries but I'll do some research to see what I can do.  Please send me any repos that were helpful, even if outdated.

**Comment by tormentous** [comment]
it has been a year whats the status?

**Comment by jasonacox** [comment]
Thanks @tormentous !   Unfortunately, I have no Android devices to use to test with yet.  I'll look at Android Studio to see if can provide enough emulation testing without an actual device to test.

**Comment by jasonacox** [comment]
Yes, I have discovered that it will require [NDK](https://developer.android.com/ndk/downloads/index.html) as well as the [Android SDK](https://developer.android.com/studio). The process is a bit gross for automation (GUI to install or more complex sequences via command line) to get it all together.  I'm working to see if I can get it to work.  

Because the build process will be *significantly* different that what we do for Apple device targets (iOS, macOS, tvOS), I am not going to add Android library generation to this project.  I will create a new project targeting Android devices if I am able to get it automated.

**Comment by tormentous** [comment]
> Yes, I have discovered that it will require [NDK](https://developer.android.com/ndk/downloads/index.html) as well as the [Android SDK](https://developer.android.com/studio). The process is a bit gross for automation (GUI to install or more complex sequences via command line) to get it all together. I'm working to see if I can get it to work.
> 
> Because the build process will be _significantly_ different that what we do for Apple device targets (iOS, macOS, tvOS), I am not going to add Android library generation to this project. I will create a new project targeting Android devices if I am able to get it automated.

thank you jason! Also, please include “executable outputs” for 32 and 64 bits architectures.
